### PR TITLE
[MIG] purchase_order_approval_block: Migration to 13.0

### PR DESCRIPTION
**Pull request by albariera**
Standard migration. No real changes.

### CODE REVIEW

File: purchase_order_approval_block/models/purchase_order.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        comodel_name="purchase.approval.block.reason", string="Approval Block Reason",
+    )
+    approval_blocked = fields.Boolean(
+        "Approval Blocked", compute="_compute_approval_blocked",
+    )
+
+    def _compute_approval_blocked(self):
+        for rec in self:
+            if rec.approval_block_id:
+                rec.approval_blocked = True
```
Comment (HviorForgeFlow) [inline purchase_order_approval_block/models/purchase_order.py]: Compute fields on version 13.0 must always return a value, add an equal False if no approval_block_id is found.

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

File: purchase_order_approval_block/models/purchase_order.py (hunk 1 above)
Comment (albariera) [reply purchase_order_approval_block/models/purchase_order.py]: Thanks, comment attended

File: purchase_order_approval_block/models/purchase_order.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        comodel_name="purchase.approval.block.reason", string="Approval Block Reason",
+    )
+    approval_blocked = fields.Boolean(
+        "Approval Blocked", compute="_compute_approval_blocked",
+    )
+
+    @api.depends("approval_block_id")
+    def _compute_approval_blocked(self):
+        for rec in self:
+            rec.approval_blocked = rec.approval_block_id
```
Comment (HviorForgeFlow) [inline purchase_order_approval_block/models/purchase_order.py:20]: Approval Blocked boolean field expects True or False
```suggestion
        for rec in self:
            if rec.approval_block_id:
                rec.approval_blocked = True
            else:
                rec.approval_blocked = False
```

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

File: purchase_order_approval_block/models/purchase_order.py (hunk 2 above)
Comment (etobella) [reply purchase_order_approval_block/models/purchase_order.py:20]: When assigning the value, it is transforming the new value to cache. So It is using the following function:
https://github.com/odoo/odoo/blob/13.0/odoo/fields.py#L1180

It is not needed.

**Review by etobella — APPROVED** [review]
LGTM :+1:

File: purchase_order_approval_block/models/purchase_order.py (hunk 2 above)
Comment (HviorForgeFlow) [reply purchase_order_approval_block/models/purchase_order.py:20]: Ok!

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**